release.nix

```diff
@@ -0,0 +1,8 @@
+{ nixpkgs ? null, ... }@_args:
+with builtins;
+let
+  names = filter (n: _args.${n} != null) (attrNames _args);
+  args = listToAttrs (map (name: { inherit name; value = _args.${name}; }) names);in
```

Comment:
This is basically taking all arguments that aren't null, correct? I you added a comment or named the variable `nonNullArgs` it would be a bit clearer to me.

You could also use `filterAttrs` https://github.com/NixOS/nixpkgs/blob/master/lib/attrsets.nix#L124.

Can you explain why that is important for hydra integration? It would be awesome if you could capture that in a comment so that the context is preserved in the future (there is no direct hint in the code why it is needed and what is needed directly to keep hydra working.

What about using https://github.com/NixOS/nixpkgs/blob/master/lib/attrsets.nix#L124 ?